ee/cis/macos-14/test/scripts/CIS_3.4.sh

```
@@ -1,25 +1,27 @@
-origExpire=$(cat ./tmp.txt  | grep expire-after);
-sed "s/${origExpire}/expire-after:60d OR 5G/" ./tmp.txt > /etc/security/audit_control;
-rm ./tmp.txt;
+AUDIT_FILE="/etc/security/audit_control"
+TMP_FILE="/tmp/audit_control.tmp"
 
+# If expire-after exists, replace it; otherwise append it.
+if /usr/bin/sudo /usr/bin/grep -q "^expire-after:" "$AUDIT_FILE"; then
+    /usr/bin/sudo /usr/bin/awk '
+        /^expire-after:/ { print "expire-after:60d OR 5G"; next }
+        { print }
+    ' "$AUDIT_FILE" > "$TMP_FILE"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)